source/guides/stacks/dars.rst

```diff
@@ -41,78 +42,137 @@ in the |CL|-based containers may not be the latest released by |CL|.
 Using the Docker images
 ***********************
 
-#. To immediately start using the latest stable DARS images, pull an image
+Launching the Image
+===================
+
+#. To use the latest stable DARS images, pull an image
    directly from `Docker Hub`_. This example uses the
    `DARS with Intel® MKL`_ Docker image.
 
+   .. code-block:: bash
+
+      docker pull clearlinux/stacks-dars-mkl
+
+
 #. Once you have downloaded the image, you can run it with
 
    .. code-block:: bash
 
       docker run -it --ulimit nofile=1000000:1000000 --name mkl <name of image>
 
-   This will launch the image and drop you into a bash shell inside the
-   container.  You will see output similar to the following:
+   This will launch the image and drop you into a bash shell inside the container. The :command:`--ulimit nofile=` parameter is required in order to increase the allowed number of open files for the Apache Spark engine.
 
-   .. code-block:: console
+   If you need to verify the name of the DARS image, you can use the :command:`docker image ls` command to see which images reside on your system.
 
-      root@fd5155b89857 /root # spark-shell
-      spark-shell
-      Config directory: /usr/share/defaults/spark/
-      Welcome to
-        ____              __
-       / __/__  ___ _____/ /__
-       _\ \/ _ \/ _ `/ __/  '_/
-      /___/ .__/\_,_/_/ /_/\_\   version 2.4.0
-         /_/
+   .. code-block:: bash
+
+      docker image ls
 
-      Using Scala version 2.12.7 (OpenJDK 64-Bit Server VM, Java 1.8.0-internal)
-      Type in expressions to have them evaluated.
-      Type :help for more information.
 
-      scala>
+   .. code-block:: console
+
+      REPOSITORY                                                   TAG                 IMAGE ID            CREATED             SIZE
+      clearlinux/stacks-dars-mkl                                   test-img            49a70a22231f        23 hours ago        2.66GB
+      ubuntu                                                       latest              2ca708c1c9cc        7 days ago          64.2MB
+      katadocker/kata-deploy                                       latest              bd6dc92f8060        7 days ago          673MB
+      clearlinux/stacks-dars-mkl                                   latest              2c9555536d5f        4 weeks ago         2.62GB
+
 
-   The :command:`--ulimit nofile` parameter is currently required in order to
-   increase the number of open files opened at certain point by the spark
-   engine.
 
 Building DARS images
-********************
+====================
 
-If you choose to build your own DARS container images, you can customize
-them as needed. Use the provided Dockerfile as a baseline.
+If you choose to build your own DARS container images, you can customize them as needed. Use the :file:`Dockerfile` included in the Github\* repository as your baseline. You can also follow :ref:`custom-app-container` for details on customizing containers on |CL|.
```

Comment:
@bd-dean. I apologize. It looks like your PR will make the finish line before the `custom-app-container.`
So for now, please remove the line :  `You can also follow...`. We can that line back in later on. 
After that's removed, this PR will pass the CI check.

Reply:
backed out, I'll add back with next update